### DISCUSSION TITLE
Using relayPagination() on nested fields

### DISCUSSION THREAD

**Discussion by oreqizer** [urql-graphql/urql#1133] [Q&A]
Hi,

my schema looks something like:

```graphql
query {
  account {
    results() { # this field is Relay pagination compatible
      edges {
        # ...
      }
    }
  }
}
```

is it possible to use `relayPagination()` on a nested field?

**Comment by kitten** [comment]
Yes, you’re not limited to adding it for any `Query.field` https://formidable.com/open-source/urql/docs/graphcache/computed-queries/#resolvers

It’s a resolver so you can add it onto any field as shown in the docs above, so `Account.results` (if that’s approximately what your type is called) works too 🙌
